tests/unit/restapi/lib/actions.py

```diff
@@ -101,3 +104,32 @@ def register_queue(
         json=payload,
         follow_redirects=True,
     )
+
+
+def create_plugin_parameter_type(
+    client: FlaskClient,
+    name: str,
+    group_id: int,
+    structure: dict[str, Any] | None = None,
+) -> TestResponse:
+    """Create a Plugin Parameter Type using the API.
+
+    Args:
+        client: The Flask test client.
+        name: The name of the plugin parameter type to be created.
+        group_id: The group to create the new plugin parameter type in.
+        structure: Optional JSON-type field for further constraining a type's structure.
+
+    Returns:
+        The response from the API.
+    """
+    payload = {"name": name, "group_id": group_id}
+
+    if structure:
+        payload["structure"] = structure
```

Comment:
> should have a `description` as well. Present in ORM and added to schema by #460
```suggestion
    payload = {"name": name, "group_id": group_id}

    if structure:
        payload["structure"] = structure
        
    if description:
        payload["description"] = description
```
Adding description to payload